### PR TITLE
Upgrading cascade-assets to rails 6

### PR DESCRIPTION
**Pull request by cbryant24**
Story: https://chapman.my.workfront.com/task/5eeaa0f600bf8b9d0b6303b5f7b2a192/updates

### CODE REVIEW

File: app/data_definitions/ad_landing.rb [hunk 1]
```diff
@@ -22,11 +22,11 @@ class AdLanding < DataDefinitions::Base
...
       # Selector => Value
       XPATH[:header_text] => 'MAKING <strong>BOLD</strong> STEPS',
-      XPATH[:desktop_logo_src] => '/_assets/ad_landing_page/desktop_logo.svg',
+      XPATH[:desktop_logo_src] => 'media/images/ad_landing_page/desktop_logo.svg',
       XPATH[:desktop_logo_alt] => 'desktop logo',
-      XPATH[:mobile_logo_src] => '/_assets/ad_landing_page/mobile_logo.svg',
+      XPATH[:mobile_logo_src] => 'media/images/ad_landing_page/mobile_logo.svg',
       XPATH[:mobile_logo_alt] => 'mobile logo',
-      XPATH[:background_image] => '/_assets/ad_landing_page/dmac-masthead.jpg',
+      XPATH[:background_image] => 'media/images/ad_landing_page/dmac-masthead.jpg',
```
Comment (cbryant24) [inline app/data_definitions/ad_landing.rb:29]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/three_column.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ class ThreeColumn < DataDefinitions::Base
     DEFAULTS = {
       XPATH[:masthead_type] => 'Branded Masthead',
       XPATH[:branded_header] => 'A Brand New Branded Masthead',
-      XPATH[:branded_image_path] => '/_assets/mastheads/branded-new.jpg',
+      XPATH[:branded_image_path] => 'media/images/mastheads/branded-new.jpg',
```
Comment (cbryant24) [inline app/data_definitions/three_column.rb:16]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/three_column.rb [hunk 3]
```diff
@@ -27,19 +27,19 @@ class ThreeColumn < DataDefinitions::Base
               subTitle: 'Optional text can go here',
               altText: 'image alt attribute here',
               photoBy: 'I.M. Photographer',
-              image: {path: '/_assets/mastheads/slider-v201611.jpg'}
+              image: {path: 'media/images/mastheads/slider-v201611.jpg'}
```
Comment (cbryant24) [inline app/data_definitions/three_column.rb:30]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/three_column.rb [hunk 4]
```diff
@@ -27,19 +27,19 @@ class ThreeColumn < DataDefinitions::Base
...
             },
             {
               subTitle: '',
               altText: 'image alt attribute here',
               photoBy: '',
-              image: {path: '/_assets/mastheads/slider-v201611.jpg'}
+              image: {path: 'media/images/mastheads/slider-v201611.jpg'}
```
Comment (cbryant24) [inline app/data_definitions/three_column.rb:36]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/three_column.rb [hunk 5]
```diff
@@ -27,19 +27,19 @@ class ThreeColumn < DataDefinitions::Base
...
             },
             {
               subTitle: 'Slider Slide #3 of 3',
               altText: 'image alt attribute here',
               photoBy: '',
-              image: {path: '/_assets/mastheads/slider-v201611.jpg'}
+              image: {path: 'media/images/mastheads/slider-v201611.jpg'}
```
Comment (cbryant24) [inline app/data_definitions/three_column.rb:42]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/two_column.rb [hunk 6]
```diff
@@ -13,7 +13,7 @@ class TwoColumn < DataDefinitions::Base
     DEFAULTS = {
       XPATH[:masthead_type] => 'Branded Masthead',
       XPATH[:branded_header] => 'A Brand New Branded Masthead',
-      XPATH[:branded_image_path] => '/_assets/mastheads/branded-new.jpg',
+      XPATH[:branded_image_path] => 'media/images/mastheads/branded-new.jpg',
```
Comment (cbryant24) [inline app/data_definitions/two_column.rb:16]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/two_column.rb [hunk 7]
```diff
@@ -28,19 +28,19 @@ class TwoColumn < DataDefinitions::Base
               subTitle: '',
               altText: 'the altText is not the alt-right',
               photoBy: 'I.M. Photographer',
-              image: {path: '/_assets/mastheads/slider-v201611.jpg'}
+              image: {path: 'media/images/mastheads/slider-v201611.jpg'}
```
Comment (cbryant24) [inline app/data_definitions/two_column.rb:31]: Images are now served from the updated webpacker assets pipelin

File: app/data_definitions/two_column.rb [hunk 8]
```diff
@@ -28,19 +28,19 @@ class TwoColumn < DataDefinitions::Base
...
             },
             {
               subTitle: 'Optional text can go here',
               altText: 'image alt attribute here',
               photoBy: '',
-              image: {path: '/_assets/mastheads/slider-v201611.jpg'}
+              image: {path: 'media/images/mastheads/slider-v201611.jpg'}
```
Comment (cbryant24) [inline app/data_definitions/two_column.rb:37]: Images are now served from the updated webpacker assets pipeline

File: app/data_definitions/two_column.rb [hunk 9]
```diff
@@ -28,19 +28,19 @@ class TwoColumn < DataDefinitions::Base
...
             },
             {
               subTitle: 'Slider Slide \n #3 of 3',
               altText: 'image alt attribute here',
               photoBy: '',
-              image: {path: '/_assets/mastheads/slider-v201611.jpg'}
+              image: {path: 'media/images/mastheads/slider-v201611.jpg'}
```
Comment (cbryant24) [inline app/data_definitions/two_column.rb:43]: Images are now served from the updated webpacker assets pipeline

**Comment by nicholasnadel** [comment]
@cbryant24 `bin/build` errors out for me at the end and doesn't build  `dist/env/cascade-assets.xml` 
see https://gist.github.com/nicholasnadel/cda6260fabbaae753352507814574ee1

**Comment by corejh** [comment]
I'm failing when I go to start up the rails server locally. Specifically

`17:55:18 webpacker.1 | warning package.json: No license field`
`17:55:18 webpacker.1 | error Command "webpack-dev-server" not found.`

Is there something specific that I need to be doing? The only instruction I saw was to set the node environment variable which I've done

`johoang@SMC-JOHOANG-MBP cascade-assets % printenv`
`...`
`NODE_ENV=development`

**Comment by cbryant24** [comment]
@corejh sorry forgot to include you need to run the command `npm install`

**Comment by nicholasnadel** [comment]
@cbryant24 @corejh 

FYI, the following resolved NPM issues for me:
- `rm -rf node_modules`
- manually remove the font awesome line from `package.json` (not sure why deleting didn't work)
- `bundle exec rake serve`

now `modular/whatever`, `bin/build` and everything else is working as expected

**Comment by corejh** [comment]
@cbryant24 Can you add the steps Nick posted to the documentation, otherwise it's looking good here

**Pull request merged**